test/operand_capabilities_test.cpp

```diff
@@ -469,6 +498,13 @@ INSTANTIATE_TEST_SUITE_P(
                 CASE1(DECORATION, Decoration::Alignment, Kernel),
             })));
 
+// See SPIR-V Section 3.20 Decoration
+INSTANTIATE_TEST_SUITE_P(Decoration_1_6, EnumCapabilityTest,
+                         Combine(Values(SPV_ENV_UNIVERSAL_1_6),
+                                 ValuesIn(std::vector<EnumCapabilityCase>{
+                                     CASE2(DECORATION, Decoration::Uniform,
+                                           Shader, UniformDecoration)})));
```

Comment:
The test starting at line 449 checks that in SPIR-V 1.0 and 1.1  the "Uniform" decoration is deemed enabled by Shader.
But this new test shows that in 1.6 the Uniform decoration is also enabled by UniformDecoration, which was introduced in 1.6